test/conftest.py

```diff
@@ -392,54 +381,36 @@ def pudl_tmpdir(tmp_path_factory):
     return tmpdir
 
 
-@pytest.fixture(scope="session")
-def pudl_output_tmpdir(pudl_tmpdir):
-    tmpdir = pudl_tmpdir / "output"
-    tmpdir.mkdir()
-    return tmpdir
-
-
-@pytest.fixture(scope="session")
-def pudl_input_tmpdir(pudl_tmpdir):
-    tmpdir = pudl_tmpdir / "data"
-    tmpdir.mkdir()
-    return tmpdir
+def pytest_sessionstart(session):
+    """Configures input/output paths for the tests."""
+    # TODO(rousik): Should we be using fixed paths instead
+    # of using tmpdir capabilities here?
+    pudl.workspace.setup.set_path_overrides(
+        input_dir="~/pudl-work/data",
+        output_dir="~/pudl-work/output",
+    )
```

Comment:
Why are hard-coded Unix paths a good idea here?  Don't we want the test outputs to go to a pytest temporary directory for isolation from the user's environment?

Reply:
Uh, indeed. I must have conflated this with the `READTHEDOCS` case. Will also fix.